cmd/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"context"
+	"github.com/integrity-sum/internal/configs"
+	logConfig "github.com/integrity-sum/pkg/logger"
+	"log"
+	"os"
+	"os/signal"
+
+	"github.com/integrity-sum/internal/initialize"
+	"github.com/joho/godotenv"
+)
+
+func main() {
+	// Load values from .env into the system
+	if err := godotenv.Load(); err != nil {
+		log.Print("No .env file found")
```

Comment:
should we stop executing if the .env file was not found?

Reply:
Ok. Understood. Fixed it